### ISSUE TITLE
Ran out of reserved memory error when launching dota 2

### ISSUE THREAD

**Issue by cc1w** [ValveSoftware/Dota2-Gameplay#14144]
### Description

![error dota](https://github.com/ValveSoftware/Dota2-Gameplay/assets/153728506/9b06d94e-3683-40bf-829a-c815c83d551b)

### Screenshots

![error dota](https://github.com/ValveSoftware/Dota2-Gameplay/assets/153728506/69465dbd-a5fd-44ea-a06b-05db0cc3f330)

**Comment by commentator24** [comment]
clear your cache, disk clean up and defrag #14138

**Comment by cc1w** [comment]
how do i do that ? can u explain pls ,!
!

**Comment by commentator24** [comment]
i already explained in other thread. 
or you can just google how to clear cache
hope it works for you.

**Comment by amirmkp** [comment]
did you find a way to solve the problem?

**Comment by milindpatel63** [comment]
i'm still having this issue. Kindly fix this. 
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/11052427/4622d7d2-3a12-4f82-89ef-254283c1a31a)

**Comment by cc1w** [comment]
> [milindpatel63](/milindpatel63)

yes i fixed this by instaling the game again in the second Hard disk ( changed the game from Disk C to D )
